src/mac.m

```diff
@@ -1767,7 +1772,7 @@ - (void)windowDidExitFullScreen:(NSNotification*)notification
 
   [impl->wrapperView setFrame:framePt];
   [impl->drawView setFrame:sizePt];
```

Comment:
Now I'm confused, calling setFrame (not `setFrameSize`) on an NSView here, not an NSWindow, which doesn't even seem to be a method.  Although I'm just about to nuke `puglSetFrame` entirely (combining size and position is the devil, both for Wayland, and in general due to embedding and scaling bugs), so I guess this particular case doesn't matter anyway.

Reply:
nsviews do have setFrame, as they have position and size. subviews can be offset from their parent